src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -112,27 +115,27 @@ public static ConnectionBuilder<TSourceType> Create<TNodeType>(string name = "de
                 Type = typeof(TConnectionType),
                 Arguments = new QueryArguments(),
             };
-            return new ConnectionBuilder<TSourceType>(fieldType, false, false, null)
-                .Unidirectional();
+            fieldType.Arguments.Add(new QueryArgument(typeof(StringGraphType))
+            {
+                Name = "after",
+                Description = "Only look at connected edges with cursors greater than the value of `after`.",
+            });
+            fieldType.Arguments.Add(new QueryArgument(typeof(IntGraphType))
+            {
+                Name = "first",
+                Description = "Specifies the number of edges to return starting from `after` or the first entry if `after` is not specified.",
```

Comment:
Seems a bit weird. https://relay.dev/graphql/connections.htm#sec-Forward-pagination-arguments

```suggestion
                Description = "Specifies the number of edges to return (at most) starting from `after` or just the first number of edges if `after` is not specified.",
```

Reply:
This change will affect existing schemas.  Is this okay?  If not, we should only make this change for the added `ConnectionBuilder_Typed.cs`.  Note that we will be deleting the untyped versions in v5.

Reply:
Affect how? This is just  description.

Reply:
Unlike xml comments, the description is part of the generated schema. I understand there is no functionality difference.

Reply:
```suggestion
                Description = "Specifies the maximum number of edges to return, starting after the cursor specified by `after`, or the first number of edges if `after` is not specified.",
```

Reply:
I would suggest that we are okay making schema changes of this nature for new minor versions.  (e.g. 4.3.0, but not 4.2.2)  Since this will be a new minor version anyway (new API), I would say this is fine to merge.

Reply:
ok